doc/release-notes/RELEASE-0.8.md

```diff
@@ -0,0 +1,60 @@
+
+# Release v0.8
+
+We are pleased to announce the release of Screed v0.8. Screed is a database
+engine capable of storing and retrieving short-read sequence data. Screed is
+designed to be fast and adaptable to different sequence file formats. This 
+release of Screed we consider to be stable and complete.
+
+This version of Screed contains developer documentation for contributing to the
+Screed project and a code of conduct for interacting with other contributors
+and project maintainers. Documentation is available at
+http://screed.readthedocs.org/en/v0.8/ 
+
+## New items of note:
+
+This release successfully installs and passes its unit tests on Ubuntu 14.04 
+and the latest release of Mac OS X 10 "Yosemite". It also passes the khmer 
+acceptance tests as per the [eelpond testing protocol.]
+(https://github.com/ged-lab/literate-resting/blob/master/kp/README.txt)
+
+ - Screed now has automatic compression detection via magic bit sniffing gzip, 
+ bzip and zip files(from @mr-c)
+ - Screed now supports streaming of uncompressed FASTA and FASTQ files. Bzip2 
+ files can also be streamed but not gzip files. ([from @mr-c]
+ (https://github.com/ged-lab/screed/pull/11))
+ - Screed now has a Changelog, developer documentation and uses the 'quality' 
+ attribute over the 'accuracy' attribute in dealing with FASTQ reads. ([from 
```

Comment:
The accuracy-> quality change is quite a big deal as it changes the API. I would move it up to be the 2nd paragraph of the release notes. To be more precise I would say: "Screed has renamed the 'accuracy' attribute of read records to the name 'quality'; this API change will need to be adopted by all users who want to upgrade to this version. Unlike the khmer project, Screed does not yet adhere to semantic versioning. That will come with the 1.0 release"